packages/components/src/TitleBlockZen/types.ts

```diff
@@ -157,7 +157,7 @@ export type SectionTitleRenderProps = Pick<
   | 'sectionTitleDescriptionAutomationId'
 >
 
-export type TitleBlockVariant = 'admin' | 'education' // the default is wisteria bg (AKA "reporting")
+export type TitleBlockVariant = 'admin' | 'education' | 'home' // the default is wisteria bg (AKA "reporting")
```

Comment:
Can we also have a type description here to call out that the `home` variant should not be used outside of the home page context. Just so we have some intellisense to dissuade folks from using this.

Also tiny clean up opportunity here, can you please remove the code comment:
`// the default is wisteria bg (AKA "reporting")`

I think wisteria was from the bygone greycliff theme era.